README.md

```diff
@@ -1,77 +1,31 @@
-# Populating an excel sheet with Gspread library
+# GSpread and Weather API Integration - Pytest Test Suite
 
-## Overview
-
-This project allows you to upload the contents of a Pandas DataFrame to a Google Sheet using the gspread library. Specifically, it checks if a worksheet with a specified title exists in the spreadsheet. If the worksheet doesn't exist, it creates one. It then populates the worksheet with the headers and values from the DataFrame.
-
-
-## Tools Used
-The following tools were used in this project:
-   - Python
-   - Gspread library
-   - Pandas library
-   - Dotenv
-
-##  Data Source
-The data used was the first 500 rows of a dataset obtained from Kaggle. click on this [link](https://www.kaggle.com/datasets/lissetteg/ecommerce-dataset) to access the file.
+This repository contains a pytest test suite for testing the upload of a json file from weatherAPI to a Google Sheet using the gspread library.
 
 ## Prerequisites
 
-Before using this script, ensure you have the following prerequisites in place:
```

Comment:
Why did you delete your former readme description?

Reply:
oh i didnt delete it, it has already been merged with the main branch after you approved it, this is probably because i edited a copy of it for this new readme sir